src/dotnet-core-uninstall/Shared/VSVersioning/VSVersionHelper.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo.Versioning;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Exceptions;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.VSVersioning
+{
+    /// <summary>
+    /// Extracts VS versions and determine which sdk's are safe to uninstall
+    /// </summary>
+    internal static class VSVersionHelper
```

Comment:
Try to reduce comments as possible. Use method name and code structure itself to convey the info of the comments until it is too complex to do so.

Usually BlahHelper contains no extra information. "Helper" part can just be removed. In this case, let's try to condense the comment to name. VisualStudioSafeVersionsExtracter ? It might be useful to point to Kathleen's post on why it is not safe. I'll find the post later